core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -112,10 +112,10 @@ public boolean getRequiresConfirmation() {
     }
 
     /**
-     * @return permission required for user to access this management link, in addition to {@link Jenkins#ADMINISTER}
+     * @return permission required for user to access this management link, default is {@link Jenkins#ADMINISTER}
```

Comment:
AFAICT it is a breaking change in API. The default behavior changes. I doubt we care much after the removal of dangerous permissions, but it would be great to have an explicit usage abalysis

Reply:
This is actually just a change correcting a mistake in the javadoc.  The method itself originally returned `null` (I presume to help ensure subclasses actually provided a specific permission), and a access to a `ManagementLink` is controlled by a single permission.

Reply:
I am going to look for usages in plugins.
A least in Jenkins core scope, this is not a breaking change as this "getRequiredPermission()" is only used to show or not show an item in the "/manage" page. And that page is right now only visible for ADMINISTER. so there is no change there for a user with overall/READ for example.

Reply:
I will try to make it clearer on the JEP also.

Reply:
@oleg-nenashev do you have a nice example of usage analysis ? I don't know what it could looks likes.

Reply:
The JEP related description is here https://github.com/jenkinsci/jep/pull/269

Reply:
Hiding this change behind the feature flag could be useful